app/src/main/res/layout/item_follow_request.xml

```diff
@@ -80,31 +81,39 @@
         style="@style/TuskyImageButton"
         android:layout_width="52dp"
         android:layout_height="48dp"
-        android:layout_centerVertical="true"
         android:layout_marginStart="12dp"
         android:background="?attr/selectableItemBackgroundBorderless"
         android:contentDescription="@string/action_reject"
         android:padding="4dp"
         app:layout_constraintBottom_toBottomOf="@id/avatar"
         app:layout_constraintEnd_toStartOf="@id/acceptButton"
         app:layout_constraintStart_toEndOf="@id/displayNameTextView"
-        app:layout_constraintTop_toTopOf="@id/avatar"
+        app:layout_constraintTop_toBottomOf="@+id/notificationTextView"
         app:srcCompat="@drawable/ic_reject_24dp" />
 
     <ImageButton
         android:id="@+id/acceptButton"
         style="@style/TuskyImageButton"
         android:layout_width="52dp"
         android:layout_height="48dp"
-        android:layout_centerVertical="true"
-        android:layout_marginStart="12dp"
         android:layout_marginEnd="4dp"
         android:background="?attr/selectableItemBackgroundBorderless"
         android:contentDescription="@string/action_accept"
         android:padding="4dp"
-        app:layout_constraintBottom_toBottomOf="@id/avatar"
         app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintTop_toTopOf="@id/avatar"
+        app:layout_constraintTop_toTopOf="@+id/rejectButton"
         app:srcCompat="@drawable/ic_check_24dp" />
 
+    <TextView
+        android:id="@+id/account_note"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="4dp"
+        android:textSize="?attr/status_text_medium"
+        android:textIsSelectable="true"
+        app:layout_constraintEnd_toEndOf="@+id/acceptButton"
+        app:layout_constraintStart_toStartOf="@+id/usernameTextView"
+        app:layout_constraintTop_toBottomOf="@+id/rejectButton"
+        tools:text="Account note" />
```

Comment:
I'd add `android:lineSpacingMultiplier="1.1"` and `android:hyphenationFrequency="full"` like in other places, makes longer text look nicer

Reply:
Done.